tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -364,6 +364,58 @@ public void ShowSchedule_PlayoffScheme_RoundNamesAreCreated()
             CollectionAssert.AreEqual(actual.RoundNames, expected.RoundNames);
         }
 
+        /// <summary>
+        /// Test for ShowSchedule method.
+        /// Valid schedule is passed, no exception occurred.
+        /// </summary>
+        [TestMethod]
+        public void ShowSchedule_PlayoffScheme_TeamsNamesAreAssigned()
+        {
+            // Arrange
+            const int TEST_ROUND_COUNT = 6;
+            var tournament = CreateTournamentData(TEST_ROUND_COUNT);
+            tournament.Scheme = TournamentSchemeEnum.PlayOff;
+
+            SetupGetScheduleInfo(
+                TEST_TOURNAMENT_ID,
+                tournament);
+            SetupGetTournamentResults(
+                TEST_TOURNAMENT_ID,
+                new GameServiceTestFixture().TestGamesWithNoNamesForPlayoffRounds(TEST_ROUND_COUNT).Build());
+            
+            var expectedTeamsNames = new List<string>
+            {
+                // Every game in first round
+                "Team 1",
+                "Team 2",
+                // Bronze game
+                "Looser61",
+                "Looser62",
+                // Final game
+                "Winner61",
+                "Winner62" 
+            };
+            var actualTeamsNames = new List<string>();
+
+            var sut = BuildSUT();
+
+            // Act
+            var actual = TestExtensions.GetModel<ScheduleViewModel>(sut.ShowSchedule(TEST_TOURNAMENT_ID));
+            var rounds = actual.Rounds.Values.ToList();
```

Comment:
what if actual is null? will you be able to understand what was the reason?

Reply:
Can it possibly be null?

Reply:
Than test must fail..so it's ok I suppose.